src/components/button-bar/button-bar.test.tsx

```diff
@@ -61,10 +61,11 @@ describe("When ButtonBar children are Button components", () => {
     });
 
     buttonChildren.forEach((button) => {
-      expect(button).toHaveStyle({
-        fontSize: "var(--fontSizes100)",
-        minHeight: "40px",
-      });
+      expect(button).toHaveStyleRule("min-height", "40px");
+    });
+
+    buttonChildren.forEach((button) => {
+      expect(button).toHaveStyleRule("font-size", "var(--fontSizes100)");
```

Comment:
**question**: are both `toHaveStyleRule` checks meant to be within the `forEach` here?

Reply:
Ah my mistake. The first one was meant to be an ordinary `toHaveStyle` as it doesn't use tokens. I'll get it amended.